src/main/kotlin/racingcar/controller/RacingCarController.kt

```diff
@@ -1,25 +1,26 @@
 package racingcar.controller
 
+import racingcar.domain.NumberGenerator
 import racingcar.domain.RacingCarGame.Companion.from
 import racingcar.ui.enterCarNames
 import racingcar.ui.enterNumberOfAttempts
 import racingcar.ui.printPosition
 import racingcar.ui.printStart
 import racingcar.ui.printWinner
-import racingcar.util.RandomGenerator
+import kotlin.random.Random
 
 class RacingCarController {
 
     fun start() {
         val carNames = enterCarNames().trim()
         val numberOfAttempts = enterNumberOfAttempts()
-        val numberGenerator = RandomGenerator()
+        val numberGenerator = NumberGenerator { Random.nextInt(0, 10) }
 
         val racingCarGame = from(carNames)
 
         printStart()
 
-        for (i in 1..numberOfAttempts) {
+        repeat((1..numberOfAttempts).count()) {
```

Comment:
```suggestion
        repeat((numberOfAttempts) {
```
numberOfAttempts만으로도 충분하지 않을까요?